File: app/src/main/java/com/divercity/app/features/profile/selectgroups/SelectGroupActivity.java
package com.divercity.app.features.profile.selectgroups;

import android.content.Context;
import android.content.Intent;

import com.divercity.app.core.base.BaseFragment;
import com.divercity.app.core.base.BaseFragmentActivity;

/**
 * Created by lucas on 17/10/2018.
 */

public class SelectGroupActivity extends BaseFragmentActivity {

    public static Intent getCallingIntent(Context context){
        return new Intent(context, SelectGroupActivity.class);
    }

    @Override
    protected BaseFragment fragment() {
        return SelectGroupFragment.newInstance();
    }
}

File: app/src/main/java/com/divercity/android/features/groups/onboarding/SelectGroupActivity.kt
package com.divercity.android.features.groups.onboarding

import android.content.Context
import android.content.Intent
import com.divercity.android.core.base.BaseActivity
import com.divercity.android.core.base.BaseFragment

class SelectGroupActivity : BaseActivity() {

    companion object {
        fun getCallingIntent(context: Context) : Intent {
            return Intent(context, SelectGroupActivity::class.java)
        }
    }

    override fun fragment(): BaseFragment = SelectGroupFragment.newInstance()
}
